app/server/controllers/UserController.js

```diff
@@ -706,6 +706,49 @@ UserController.checkOutById = function(id, user, callback){
   callback);
 };
 
+/**
+ * [ADMIN ONLY]
+ *
+ * Make user an admin
+ * @param  {String}   userId   User id of the user being made admin
+ * @param  {String}   user     User making this person admin
+ * @param  {Function} callback args(err, user)
+ */
+UserController.makeAdminById = function(id, user, callback){
```

Comment:
Ah, the `user` argument was there for a future feature I never got around to implementing, which was for auditing admin actions.

Passing in the `user` making the person admin could be tracked on a separate line in this function so we could see who was doing what and when (like who decided to make x person admin??)

update call lgtm though :)